zulipterminal/model.py

```diff
@@ -142,7 +142,7 @@ def is_search_narrow(self) -> bool:
         return 'search' in [subnarrow[0] for subnarrow in self.narrow]
 
     def set_found_newest(self, found_newest: bool) -> None:
```

Comment:
Is this name okay? I think its implicitly understood that found_newest is updated per narrow. Or should I name this `set_found_newest_for_current_narrow()`?